service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/AllocationDecisionController.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import io.swagger.annotations.Api;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Api
+@RestController
+@RequestMapping("/callback/allocation-decision")
+public class AllocationDecisionController {
+
+    @PostMapping("/about-to-start")
+    @SuppressWarnings("unchecked")
```

Comment:
Again here, considering using CaseData...

Reply:
Happy to discuss, not sure what we can achieve what we tried to achieve here. It's actually from FPLA-60

Reply:
Done!

Reply:
Good changes